advanced-integration/v2/server/python/server.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import os
+
+from dotenv import load_dotenv
+from flask import Flask, request
+from paypalserversdk.http.auth.o_auth_2 import ClientCredentialsAuthCredentials
+from paypalserversdk.logging.configuration.api_logging_configuration import LoggingConfiguration, \
+    RequestLoggingConfiguration, ResponseLoggingConfiguration
+from paypalserversdk.paypalserversdk_client import PaypalserversdkClient
+from paypalserversdk.controllers.orders_controller import OrdersController
+from paypalserversdk.models.amount_with_breakdown import AmountWithBreakdown
+from paypalserversdk.models.checkout_payment_intent import CheckoutPaymentIntent
+from paypalserversdk.models.order_request import OrderRequest
+from paypalserversdk.models.purchase_unit_request import PurchaseUnitRequest
+from paypalserversdk.api_helper import ApiHelper
+
+app = Flask(__name__)
+
+load_dotenv()
```

Comment:
I think we can remove the dotenv reference as the README steps for other languages direct the user to explicitly add the environment variables in their shell. 

ref: https://github.com/paypal-examples/docs-examples/blob/c964b7083a62e76886251f34dd822fb02258b3c6/advanced-integration/v2/server/dotnet/README.md#running-the-sample

Reply:
Removed the .env files and update the README to match.  

I also caught at the last moment that Integration appears to be misspelled as `Integartion` in all the server README files.  I've corrected it here for the new additions which were based on the existing READMEs, but thought I'd point that out.

Reply:
Thanks @LeonarthusMectus, It has been fixed in other languages